teambit.bvm/list/gcp/gcp-list.ts

```diff
@@ -18,6 +18,7 @@ export enum ReleaseTypeFilter {
   DEV = 'dev',
   NIGHTLY = 'nightly',
   STABLE = 'stable',
+  ALL = 'all'
```

Comment:
I would suggest to not add new fields to this type. Instead create a new function like listAll() or pass an option to the function.

Reply:
IMO 'ALL' is a good filter. You don't always want to filter for a specific type of release so this leaves the option to pass an enum that returns all release types